packages/angular/src/generators/application/lib/update-e2e-project.ts

```diff
@@ -1,13 +1,20 @@
-import type { Tree } from '@nrwl/devkit';
+import type { ProjectConfiguration, Tree } from '@nrwl/devkit';
 import type { NormalizedSchema } from './normalized-schema';
 
-import { getWorkspacePath, offsetFromRoot, updateJson } from '@nrwl/devkit';
+import {
+  offsetFromRoot,
+  updateJson,
+  addProjectConfiguration,
+  readProjectConfiguration,
+  updateProjectConfiguration,
+  ProjectType,
+} from '@nrwl/devkit';
 import { Linter } from '@nrwl/linter';
 
-export function updateE2eProject(host: Tree, options: NormalizedSchema) {
+export function updateE2eProject(tree: Tree, options: NormalizedSchema) {
```

Comment:
nit: the other util functions for the app generator use `host`. Do we want to maintain consistency or should we start moving to using `tree`?

Reply:
Please start using `tree`, not `host` :+1: